content/learning-paths/servers-and-cloud-computing/cca-container/hello-realm.md

```diff
@@ -0,0 +1,132 @@
+---
+# User change
+title: "Run an application in a Realm"
+
+weight: 3 # 1 is first, 2 is second, etc.
+
+# Do not modify these elements
+layout: "learningpathall"
+---
+
+## Before you begin
+
+Make sure you have downloaded the [`armswdev/aemfvp-cca-image`](https://hub.docker.com/r/armswdev/aemfvp-cca-image) docker container.
+
+Install [GCC](/install-guides/gcc/) on your machine. Dependent on the architecture of your machine, you will need to install either the native compiler or cross-compiler.
+
+## Overview
+
+To run an application inside a Realm, you will package it as the "init" process for the guest linux kernel. 
+
```

Comment:
Again, I think there is a risk of misinterpretation with this wording. "Run an application inside a Realm" could be taken to mean that the realm is created for the app, whereas actually of course the whole guest VM is already the realm, and the app is just an app that inherits the confidential protection of the guest VM.

I think it would also be good to make it clear that using `init` is just a convenient pattern for this learning path experiment, just so readers are not left thinking that realm apps need to run in this way.

Reply:
Good point. Added some explicit wording around this in the Overview section.